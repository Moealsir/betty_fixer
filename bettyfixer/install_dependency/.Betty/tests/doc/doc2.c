/**
 * op_add - Do a sum
 * @arg1: First operand
 *
 * Return: Sum of two operands
 */
int op_add(int arg1, int arg2)
{
	return (arg1 + arg2);
}

/**
 * main - Program entry point
 * @av: Arguments
 *
 * Return:  0 on success. Error code otherwise
 */
int main(int ac, char **av)
{
	(void)ac;
	(void)av;
	op_add(1, 2);
	return (0);
}
