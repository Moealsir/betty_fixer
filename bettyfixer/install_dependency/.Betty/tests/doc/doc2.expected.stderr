tests/doc/doc2.c:7: warning: No description found for parameter or member 'arg2'
tests/doc/doc2.c:18: warning: No description found for parameter or member 'ac'
